demo/b2share_demo/cli.py

```diff
@@ -147,3 +149,28 @@ def import_v1_data(verbose, download, token,
     for d in dirlist:
         process_v1_record(d, indexer, base_url, logfile, verbose)
     logfile.close()
+    
+@demo.command()
+@with_appcontext
+@click.argument('base_url')
+def generate_pid_migrator(base_url):
+    url = base_url + "api/records"
+    params = {}
+    params['size'] = 1000
+    params['page'] = 1
+    response = requests.get(url,params)
+    recs = json.loads(response.text)['hits']['hits']
+    for rec in recs:
+        url_value = rec['links']['self'].replace("/api/records","/records")
+        if 'alternate_identifiers' in rec['metadata'].keys():
+            alt_ids = rec['metadata']['alternate_identifiers']
+            epic_url = None
+            for aid in alt_ids:
+                if aid['alternate_identifier_type'] == 'ePIC_PID':
+                    epic_url = aid['alternate_identifier']
+            if not(epic_url is None):
+                curl_comm = "curl -X PUT -v -H 'Accept:application/json'"
+                curl_data = '{"type":"URL","parsed_data":"%s"}' % url_value
+                curl_comm += "--data ='[%s] %s" % (curl_data, epic_url)
```

Comment:
We also need to add the PID server authentication, taken from the configuration file:
````
# CFG_EPIC_USERNAME
# CFG_EPIC_PASSWORD
# CFG_EPIC_BASEURL = 'https://epic4.storage.surfsara.nl/v2_A/handles/'
````

so add `'-u {}:{} '.format(username, pass)'`